### PR TITLE
Publish multi-arch base images (base-default & base-distroless)

### PR DESCRIPTION
**Pull request by morlay**
Signed-off-by: Morlay <morlay.null@gmail.com>

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[x] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[x] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Comment by morlay** [comment]
@howardjohn 

Since https://github.com/istio/istio/pull/28686 merged. 
We could publish multi-arch base images (`gcr.io/istio-release/base`, `gcr.io/istio-release/distroless`) first. 
arch list same as `gcr.io/istio-release/iptables`

This will be usefull for downstream to build other arch images before istio official multi-arch support.

**Comment by howardjohn** [comment]
Ah I didn't see this and opened https://github.com/istio/istio/pull/29200 doing the same...

File: tools/build-base-images.sh [hunk 1]
```diff
@@ -21,5 +21,6 @@ set -ex
 
 HUB="${HUB:-istio.io/docker}"
 TAG="${TAG:?specify a tag}"
+PLATFORMS=linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
 
-BUILDX_BAKE_EXTRA_OPTIONS="--no-cache --pull" DOCKER_TARGETS="docker.base docker.distroless docker.app_sidecar_base_debian_9 docker.app_sidecar_base_debian_10 docker.app_sidecar_base_ubuntu_xenial docker.app_sidecar_base_ubuntu_bionic docker.app_sidecar_base_ubuntu_focal docker.app_sidecar_base_centos_7 docker.app_sidecar_base_centos_8" make dockerx.pushx
+BUILDX_BAKE_EXTRA_OPTIONS="--no-cache --pull --set=base-default.platform=${PLATFORMS} --set=base-distroless.platform=${PLATFORMS}" DOCKER_TARGETS="docker.base docker.distroless docker.app_sidecar_base_debian_9 docker.app_sidecar_base_debian_10 docker.app_sidecar_base_ubuntu_xenial docker.app_sidecar_base_ubuntu_bionic docker.app_sidecar_base_ubuntu_focal docker.app_sidecar_base_centos_7 docker.app_sidecar_base_centos_8" make dockerx.pushx
```
Comment (howardjohn) [inline tools/build-base-images.sh:26]: I think we actually want `base-default` and `distroless-default`. 

Basically the problem is we have two base images (base and distroless). Then we setup variants of all of our build images, and append -base or -distroless. But for some reason, we are also appending the variant to the base images, which doesn't seem right.

Comment (morlay) [reply tools/build-base-images.sh:26]: Yeah. I make a mistake.  

I prefer #29200 did. so i closed this PR

**Pull request closed without merging**